### DISCUSSION TITLE
How to disable post-processing when loading model?

### DISCUSSION THREAD

**Discussion by ChivenZhang** [assimp/assimp#4684] [Q&A] [answered]
![`%UH}7FD JL8R_ZW``D7(XM](https://user-images.githubusercontent.com/48938612/184294453-ba57a7dd-869f-4f3a-8492-f519602070cf.png)
I need to load model from fbx file which is used to build soft body physics. When I load it which contains around 3000 vertices, finally I got 10000+ vertices which leads to incorrect simulation. Please help me, I want to load it without any post processing, How can I do ?

**Comment by kimkulling** [comment, marked as answer]
As far as I know setting the flag to 0 shall be enough to disable all post-processing steps.
